kernel/src/vm/vmar/vm_mapping.rs

```diff
@@ -283,6 +297,47 @@ impl VmMapping {
         }
     }
 
+    fn handle_page_faults_around(&self, page_fault_addr: Vaddr) -> Result<()> {
+        const AROUND_PAGE_NUM: usize = 16;
+        const AROUND_PAGE_MASK: usize = !(AROUND_PAGE_NUM * PAGE_SIZE - 1);
```

Comment:
"Around" is an adverb. Here I suggest "`SURROUNDING_PAGE_ADDR_MASK`" as an adjective.